From 7db4812f758b30d46d9dd22326149c450b807c15 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Thu, 12 Sep 2019 07:03:56 +0200
Subject: [PATCH] ENH: Adding missing include of SlicerCheckModuleEnable

When Slicer_SUPERBUILD=OFF this file does not get included and it is
required in CMake/SlicerBlockInstallCmakeProjects.cmake
---
 CMake/SlicerBlockInstallCMakeProjects.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMake/SlicerBlockInstallCMakeProjects.cmake b/CMake/SlicerBlockInstallCMakeProjects.cmake
index 8071cdb4c..a472eb916 100644
--- a/CMake/SlicerBlockInstallCMakeProjects.cmake
+++ b/CMake/SlicerBlockInstallCMakeProjects.cmake
@@ -1,3 +1,4 @@
+include(${Slicer_CMAKE_DIR}/SlicerCheckModuleEnabled.cmake)
 
 # -------------------------------------------------------------------------
 # Install VTK
-- 
2.21.0

